src/Components/Components/src/NavigationException.cs

```diff
@@ -16,6 +16,14 @@ public NavigationException(string uri)
         Location = uri;
     }
 
+    /// <summary>
+    /// Initializes a new <see cref="NavigationException"/> instance with a custom message.
+    /// </summary>
+    public NavigationException(string uri, string message) : base(message)
+    {
+        Location = uri;
+    }
+
```

Comment:
We don't need the extra constructor; we can simply pass the message to the base class since it's a static message. There's no need for the message to be customizable.

Reply:
Removed the extra constructor, static message is passed to base class